flopy/mf6/utils/postprocessing.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-from .binarygrid_util import MfGrdFile
+from flopy.mf6.utils.binarygrid_util import MfGrdFile
```

Comment:
Is there a reason you switched from an absolute path to a relative path import here?  This isn't consistent with our conventions.  Should probably switch this back.